test/lib/slots.test.ts

```diff
@@ -0,0 +1,56 @@
+import getSlots from '@lib/slots';
+import {it, expect} from '@jest/globals';
```

Comment:
Codacy has a fix for the issue: [Replace `it,·expect}·from·'@jest/globals'` with `·it,·expect·}·from·"@jest/globals"`](https://app.codacy.com/gh/calendso/calendso/pullRequest?prid=7689306)
```suggestion
import { it, expect } from "@jest/globals";
```